indexBasecodeANSWER1c.html

```diff
@@ -0,0 +1,157 @@
+</head>
+<body>
+      <br>
+      <div id="wrapper">
+            <div id="mainphoto">&nbsp;</div>  <!-- end mainphoto -->
+            <div id="nav">
+                  <table id='table1' width='900'>
+                  <tr>
+                  <td align='center' width='110' >
+                  <img src='images\mars.jpg' alt='This is Mars' height='100' id='mars'>
+					  </td>
+                  <td align='center' width='110' >
+                  <img src='images\earth.jpg' alt='This is Earth' height='100' id='earth' >
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\pluto.jpg' alt='This is Pluto' height='100' id='pluto' >
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\mercury.jpg' alt='This is Mercury' height='100' id='mercury'>
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\venus.jpg' alt='This is Venus' height='100' id='venus' >
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\moon.jpg' alt='This is Moon' height='100' width='100' id='moon' >
+                  </td>
+                  </tr>
+                  <tr>
+                  <td id='marsT' align='center' width='110' >&nbsp;</td>
+                  <td id='earthT' align='center' width='110' >&nbsp;</td>
+                  <td id='plutoT' align='center' width='110' >&nbsp;</td>
+                  <td id='mercuryT' align='center' width='110' >&nbsp;</td>
+                  <td id='venusT' align='center' width='110' >&nbsp;</td>
+                  <td id='moonT' align='center' width='110' >&nbsp;</td>
+                  </tr>
+                  </table>
+            </div> <!-- end nav -->
+            <div id="col1">
+                  <div id="col1content">
+                        <h3 id='orderform'>Flights to Mars - Order Form</h3>
+                        <p id='details'>
+                        <form method=post action="jcform.asp" name="edgar" >
+                              <p id='pt1'> First name
+                              <input type=text id="name1" ></p>
+                              <p id='pt2'> Address
+                              <input type=text id="address"  ></p>
+                              <p id='pt3'> Phone Number
+                              <input type=text id="phone" ></p>
+                              <p id='pt4'> Email
+                              <input type=text id="email" ></p>
+                              <p>Credit card type
+                              <input type=Radio name="CC" value="Visa" checked>Visa
+                              <input type=Radio name="CC" value="Amex">Amex
+                              <input type=Radio name="CC" value="Mastercard">Mastercard
+                              </p>
+                              <p>Credit card number

```

Comment:
It could be worth adding the same standards you applied above by adding an id for the Credit card number element, so that it can be identified in testing. Also to close your paragraph in the same plave on the input type feild